api/auth/model.go

```diff
@@ -71,3 +79,32 @@ type IdpClientConfig struct {
 type IDstruct struct {
 	ID string `json:"id"`
 }
+
+type Session struct {
+	ID           string    `json:"id"`
+	UserID       string    `json:"user_id"`
+	SourceID     string    `json:"source_id"`
+	Domain       string    `json:"domain"`
+	Username     string    `json:"username"`
+	RemoteAddr   string    `json:"remote_addr"`
+	UserAgent    string    `json:"user_agent"`
+	Type         string    `json:"type" jsonschema:"enum=login,enum=sso"`
+	Created      time.Time `json:"created"`
+	Updated      time.Time `json:"updated"`
+	Expires      time.Time `json:"expires"`
+	TokenExpires time.Time `json:"token_expires"`
+	LoggedOut    bool      `json:"logged_out"`
+	Current      bool      `json:"current,omitempty"`
+}
+
+type sessionsResult struct {
+	Items []Session `json:"items"`
+	Count int       `json:"count"`
+}
+
+// SearchParams search params definition
+type SearchParams struct {
+	Keywords string `json:"keywords,omitempty"`
+	UserID   string `json:"user_id,omitempty"`
+	Type     string `json:"type.omitempty" jsonschema:"enum=login,enum=sso"`
```

Comment:
Typo in "type.omitempty". A comma is needed instead of the dot.

Reply:
done